### ISSUE TITLE
CircleCI can't check out the master branch

### ISSUE THREAD

**Issue by hacklschorsch** [LeastAuthority/python-challenge-bypass-ristretto#68]
Looking for example at https://app.circleci.com/pipelines/github/LeastAuthority/python-challenge-bypass-ristretto/325/workflows/e485b40e-0c16-4c6c-8b07-077863b5b2b6/jobs/1356

Seems like we have to do the "unfollow"/"follow" dance again while being owner of the repository?  Or what helps here @btlogy ?  I think this blocks publishing wheels to pypi production.

**Comment by btlogy** [comment]
I feel like I'm going to find or writ a terraform module to just do that dance! 🕺
I'll have to do this later: I do not have the secrets w/ me.

**Comment by hacklschorsch** [comment]
> I feel like I'm going to find or writ a terraform module to just do that dance! 🕺

Lol!  I'd drop CircleCI just over the unprofessionalism they display regarding this.
Honestly, let's keep working on getting rid of this bad SaaS.

**Comment by btlogy** [comment]
I was pretty sure I've already fixed this key a few weeks ago!

For the record if this is happening again:

CircleCI shows:
![image](https://github.com/user-attachments/assets/9308314f-77ab-43c8-afd5-56a969b60a12)

GitHub shows:
![image](https://github.com/user-attachments/assets/85bbb00f-a79d-4b34-8a5e-ca54c7bb452a)

So technically, what are they missing!?

Oh, wait... @hacklschorsch: the failed build in the description is from 27th of Nov. but I've updated the key on 28th.
And the master is now successfully building (but not publishing, but that's for #67).

Or am I missing something about the checkout ?
